share/functions/__fish_config_interactive.fish

```diff
@@ -299,26 +299,6 @@ function __fish_config_interactive -d "Initializations that should be performed
         set -g fish_pager_color_description yellow
         set -g fish_pager_color_progress cyan
 
```

Comment:
Do I understand correctly that what this function used to do (per commit f71e877f18037f71c00f81e30c1173f51c42a58c ) is now handled by the builtin set_color? That seems good if so.